sql/mysqld.cc

```diff
@@ -5587,6 +5591,70 @@ static void test_lc_time_sz()
 }
 #endif//DBUG_OFF
 
+static void check_upgrade(void)
+{
+  File info_file;
+  char upgrade_from_version[1024]= {0};
```

Comment:
bit oversized here.

Reply:
I used the same size than `mysql_upgrade.cc`. If you think we should reduce this, let me know and we can include that as part of the rename PR and reduce the size in both places.
https://github.com/MariaDB/server/blob/6252a281b5020d28685777c11ee27688a9db3d66/client/mysql_upgrade.c#L74

Reply:
sure, reduce in both places.

Reply:
Does `50` sounds good?  I have used something like this: `3*len(UINT_MAX) + 2 (dots) + len('-MariaDB\n') + 3 (extra zeroes) + 6 (extra space)`

Reply:
sure